# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1479246481 -3600
#      Tue Nov 15 22:48:01 2016 +0100
# Branch stable
# Node ID 056941d5cc7ae23c52c133bb9f0cff14862b3a49
# Parent  6033e2e4d5ea8a2420c531c19d997c7baddd9c79
doc: Update month of TeX title page.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -87,17 +87,17 @@ Permission is granted to copy and distri
 manual into another language, under the above conditions for
 modified versions.
 @end ifnottex
 
 @titlepage
 @title GNU Octave
 @subtitle A high-level interactive language for numerical computations
 @subtitle Edition 4 for Octave version @value{VERSION}
-@subtitle September 2016
+@subtitle November 2016
 @sp 2
 @multitable @columnfractions 0.4 0.025 0.65
 @item
 @flushright @image{octave_logo,2.1in}
 @end flushright
 @tab
 @c this is a spacer column
 @tab
